src/System.Memory/src/System/Buffers/Binary/Reader.cs

```diff
@@ -106,12 +106,12 @@ public static T ReadMachineEndian<T>(ReadOnlySpan<byte> buffer)
 #else
             if (RuntimeHelpers.IsReferenceOrContainsReferences<T>())
             {
-                throw new ArgumentException(SR.Format(SR.Argument_InvalidTypeWithPointersNotSupported, typeof(T)));
+                ThrowHelper.ThrowArgumentException_InvalidTypeWithPointersNotSupported(typeof(T));
```

Comment:
What is the reason for this change?

Reply:
From @KrzysztofCwalina 
> If you do decide to throw, throw from a helper to allow inlining.

Previously, ThrowHelper.cs was not included for netcoreapp, but I updated the project file to always include ThrowHelper.cs

Reply:
The changes in this file should have zero effect on the inlining or the native code that will be generated for this.